orchagent/aclorch.cpp

```diff
@@ -387,14 +387,12 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
         return false;
     }
 
-    // NOTE: Temporary workaround to support matching protocol numbers on MLNX platform.
-    // In a later SAI version we will transition to using NEXT_HEADER for IPv6 on all platforms.
-    auto platform_env_var = getenv("platform");
-    string platform = platform_env_var ? platform_env_var: "";
+    // TODO: For backwards compatibility, users can substitute IP_PROTOCOL for NEXT_HEADER.
+    // This should be removed in a future release.
     if ((m_tableType == ACL_TABLE_MIRRORV6 || m_tableType == ACL_TABLE_L3V6)
```

Comment:
m_tableType == ACL_TABLE_L3V6 seems to be redundant as this should fail before reach here.

Reply:
I'm not sure I see what you mean - where would this fail?

Reply:
sorry my comment is not valid.